components/datadog/apps/fakeintake/fargateFakeintakeService.go

```diff
@@ -0,0 +1,201 @@
+					Image:       pulumi.String("public.ecr.aws/datadog/fakeintake:latest"),
+					Essential:   pulumi.BoolPtr(true),
+					MountPoints: ecs.TaskDefinitionMountPointArray{},
+					Environment: ecs.TaskDefinitionKeyValuePairArray{},
+					PortMappings: ecs.TaskDefinitionPortMappingArray{
+						ecs.TaskDefinitionPortMappingArgs{
+							ContainerPort: pulumi.Int(port),
+							HostPort:      pulumi.Int(port),
+							Protocol:      pulumi.StringPtr("tcp"),
+						},
+					},
+					VolumesFrom: ecs.TaskDefinitionVolumeFromArray{},
+				},
+			},
+			Cpu:    pulumi.StringPtr("256"),
+			Memory: pulumi.StringPtr("512"),
+			ExecutionRole: &awsx.DefaultRoleWithPolicyArgs{
+				RoleArn: pulumi.StringPtr(e.ECSTaskExecutionRole()),
+			},
+			TaskRole: &awsx.DefaultRoleWithPolicyArgs{
+				RoleArn: pulumi.StringPtr(e.ECSTaskRole()),
+			},
+			Family: e.CommonNamer.DisplayName(255, pulumi.String("fakeintake-ecs")),
+		},
+		ContinueBeforeSteadyState: pulumi.BoolPtr(true),
+	}, opts...); err != nil {
+		return nil, err
+	}
+	if e.DefaultFargateLoadBalancer() {
+		if err := e.Ctx.RegisterResourceOutputs(instance, pulumi.Map{
+			"host": alb.LoadBalancer.DnsName(),
+		}); err != nil {
+			return nil, err
+		}
+	}
+
+	return instance, nil
+}
+
+func FargateLinuxTaskDefinition(e aws.Environment, name string) (*ecs.FargateTaskDefinition, error) {
+	return ecs.NewFargateTaskDefinition(e.Ctx, e.Namer.ResourceName(name), &ecs.FargateTaskDefinitionArgs{
+		Containers: map[string]ecs.TaskDefinitionContainerDefinitionArgs{
+			containerName: *fargateLinuxContainerDefinition(),
+		},
+		Cpu:    pulumi.StringPtr("256"),
+		Memory: pulumi.StringPtr("512"),
+		ExecutionRole: &awsx.DefaultRoleWithPolicyArgs{
+			RoleArn: pulumi.StringPtr(e.ECSTaskExecutionRole()),
+		},
+		TaskRole: &awsx.DefaultRoleWithPolicyArgs{
+			RoleArn: pulumi.StringPtr(e.ECSTaskRole()),
+		},
+		Family: e.CommonNamer.DisplayName(13, pulumi.String("fakeintake-ecs")),
+	}, e.WithProviders(config.ProviderAWS, config.ProviderAWSX))
+}
+
+func fargateLinuxContainerDefinition() *ecs.TaskDefinitionContainerDefinitionArgs {
+	return &ecs.TaskDefinitionContainerDefinitionArgs{
+		Name:        pulumi.String(containerName),
+		Image:       pulumi.String("public.ecr.aws/datadog/fakeintake:latest"),
+		Essential:   pulumi.BoolPtr(true),
+		MountPoints: ecs.TaskDefinitionMountPointArray{},
+		Environment: ecs.TaskDefinitionKeyValuePairArray{},
+		PortMappings: ecs.TaskDefinitionPortMappingArray{
+			ecs.TaskDefinitionPortMappingArgs{
+				ContainerPort: pulumi.Int(port),
+				HostPort:      pulumi.Int(port),
+				Protocol:      pulumi.StringPtr("tcp"),
+			},
+		},
+		VolumesFrom: ecs.TaskDefinitionVolumeFromArray{},
+	}
+}
+
+// FargateServiceFakeintakeWithoutLoadBalancer deploys one fakeintake container to a dedicated Fargate cluster
+// Hardcoded on sandbox
+func FargateServiceFakeintakeWithoutLoadBalancer(e aws.Environment) (ipAddress pulumi.StringOutput, err error) {
```

Comment:
<!-- Questions are appropriate if you have a potential concern but are not quite sure if it's relevant or not. Asking the author for clarification or investigation can lead to a quick resolution. -->
**❓ question**
Can this be private ?